src/main/java/com/internet/shop/controllers/IndexController.java

```diff
@@ -25,9 +27,9 @@
     public void init() throws ServletException {
         productService.create(new Product("Banana", 10.00));
         productService.create(new Product("Orange", 20.00));
-        User bob = userService.create(new User("Bob", "Bob", "1"));
-        shoppingCartService.create(new ShoppingCart(bob.getId()));
-        User alice = userService.create(new User("Alice", "Alice", "2"));
+        userService.create(new User("Bob", "Bob", "1", Set.of(Role.of("ADMIN"))));
+        User alice = userService
+                .create(new User("Alice", "Alice", "2", Set.of(Role.of("USER"))));
         shoppingCartService.create(new ShoppingCart(alice.getId()));
```

Comment:
let's create a separate InjectDataController for this purpose
we should not place business logic in the init() method